packages/eslint-config-react-app/index.js

```diff
@@ -315,5 +341,37 @@ module.exports = {
     'flowtype/define-flow-type': 'warn',
     'flowtype/require-valid-file-annotation': 'warn',
     'flowtype/use-flow-type': 'warn',
+
+    // https://github.com/testing-library/eslint-plugin-jest-dom
+    ...(hasJestDom
```

Comment:
FYI the original PR only adds these rule to test files which is probably better: https://github.com/facebook/create-react-app/pull/8155/files#diff-b4a510397b084f197f379c4e3485cde1